source/user_guide.rst

```diff
@@ -20,115 +17,95 @@ Who should read this
 Getting Started
 ---------------
 
-To start, please follow dojot's installation guide in
-:doc:`install/compose_guide`. There you should find how to properly download a
-working copy of the components, how to minimally configure them, how to start
-them up and how to check whether they are working.
+To start, please follow dojot's installation guide in :doc:`install/compose_guide`. There you should find how to
+properly download a working copy of the components, how to minimally configure them, how to start them up and how to
+check whether they are working.
 
 dojot basics
 ------------
 
-Before using dojot, you should be familiar with some basic operations and
-concepts. They are very simple to understand and use, but without them, all
-operations might become obscure and senseless. It is advisable to checkout our
+Before using dojot, you should be familiar with some basic operations and concepts. They are very simple to understand
+and use, but without them, all operations might become obscure and senseless. It is advisable to checkout our
 :doc:`architecture` to get acquainted with all internal components.
 
-First of all, you should check out how to acccess dojot through its APIs, which
-is detailed in the next section. After that, there's an explanation of a few
-basic entities in dojot: devices, templates and flows (including a simple
+First of all, you should check out how to acccess dojot through its APIs, which is detailed in the next section. After
+that, there's an explanation of a few basic entities in dojot: devices, templates and flows (including a simple
 tutorial on how to create and use them).
 
-All these instructions considers only API access. For a guided tour on how to
-use the web interface, check dojot's `YouTube channel`_.
+All these instructions considers only API access. For a guided tour on how to use the web interface, check dojot's
+`YouTube channel`_.
 
 User authentication
 *******************
 
-All HTTP requests supported by dojot are sent to the API gateway. In order to
-control which user should access which endpoints and resources, dojot makes
-uses of `JSON Web Token`_ (a useful tool is `jwt.io`_) which encodes things
-like (not limited to these):
+All HTTP requests supported by dojot are sent to the API gateway. In order to control which user should access which
+endpoints and resources, dojot makes uses of `JSON Web Token`_ (a useful tool is `jwt.io`_) which encodes things like
+(not limited to these):
 
 - User identity
 - Validation data
 - Timestamp
```

Comment:
This might be misleading - If I recall correctly, what we have in a JWT is a kind of "best before" label, meaning that this token should be valid only until the provided timestamp.

Reply:
Like an expiration date?